### PR TITLE
add docker

### PR DESCRIPTION
**Pull request by upacyxou**
- docker added
- readme updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,15 +8,22 @@ This repository is to be used as proof generator for SealHub
...
 
-| Name                                     | Description                              |
-| ---------------------------------------- | ---------------------------------------- |
-| `MONGO`                                  | URL of the mongo database                |
-| `PORT`                                   | Port to run server on (defaults to 1337) |
+| Name     | Description                              |
+| -------- | ---------------------------------------- |
+| `MONGO`  | URL of the mongo database                |
+| `PORT`   | Port to run server on (defaults to 1337) |
+| `DOMAIN` | Your domain for docker                   |
```
Comment (backmeupplz) [inline README.md]: Isn't this optional?

File: README.md [hunk 2]
```diff
@@ -8,15 +8,22 @@ This repository is to be used as proof generator for SealHub
...
 
 ## Environment variables
 
-| Name                                     | Description                              |
-| ---------------------------------------- | ---------------------------------------- |
-| `MONGO`                                  | URL of the mongo database                |
-| `PORT`                                   | Port to run server on (defaults to 1337) |
+| Name     | Description                              |
+| -------- | ---------------------------------------- |
+| `MONGO`  | URL of the mongo database                |
```
Comment (backmeupplz) [inline README.md]: Isn't this optional?

File: Dockerfile.backend.production [hunk 3]
```diff
@@ -0,0 +1,7 @@
+FROM node:18
+WORKDIR /usr
+COPY . ./
+RUN yarn
+RUN yarn get-zkp
```
Comment (backmeupplz) [inline Dockerfile.backend.production]: let's do `download-zk-files` instead

File: Dockerfile.backend.development [hunk 4]
```diff
@@ -0,0 +1,7 @@
+FROM node:18
+WORKDIR /usr
+COPY . ./
+RUN yarn
+RUN yarn get-zkp
+EXPOSE 1337
```
Comment (backmeupplz) [inline Dockerfile.backend.development]: this should be done in docker-compose, I think

File: .dockerignore [hunk 5]
```diff
@@ -0,0 +1,2 @@
+dist
```
Comment (backmeupplz) [inline .dockerignore:1]: Please, don't leave files without the extra linebreak in the end

File: README.md [hunk 6]
```diff
@@ -8,15 +8,22 @@ This repository is to be used as proof generator for SealHub
 2. Launch the [mongo database](https://www.mongodb.com/) locally
 3. Create `.env` with the environment variables listed below
 4. Run `yarn` in the root folder
-5. Run `yarn start`
+5. Run `yarn gep-zkp` for all ZK files
```
Comment (avr6ude) [inline README.md]: typo
correct: `yarn get-zkp`

**Review by backmeupplz — APPROVED** [review]
lgtm

**Pull request merged**